using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md

```diff
@@ -0,0 +1,417 @@
+targetParameters[@"profile.membershipLevel"] = @"platinum";
+
+// Add product parameters
+targetParameters[@"productId"] = @"111";
+targetParameters[@"categoryId"] = @"Books";
+
+// Add order parameters
+targetParameters[@"orderId"] = @"10";
+targetParameters[@"orderTotal"] = @"110.56";
+targetParameters[@"purchasedProductIds"] = @"111";
+
+[data setObject:[NSDictionary dictionaryWithObject:targetParameters forKey:@"target"] forKey:@"__adobe"];
+
+AEPDecisionScope* decisionScope = [[AEPDecisionScope alloc] initWithName:@"myTargetLocation"]; // Target location (or mbox)
+[AEPMobileOptimize updatePropositions:@[decisionScope] withXdm:nil andData:data];
+```
+{% endtab %}
+{% endtabs %}
+
+### Target Third Party ID
+
+To use Target Third Party ID in the Experience Edge mobile workflows, the corresponding namespace needs to be configured in Experience Platform Data Collection.
+
+1. On [Experience Platform Data Collection](https://launch.adobe.com/), navigate to **Data Collection** > **Datatreams** using the left navigation panel.
+2. Select your configured datastream and click on the desired environment from the list.
+3. Navigate to **Adobe Target** section, specify the **Target Third Party ID Namespace**.
+4. Click **Save**.
+
+![Target Third Party ID configuration](../../.gitbook/assets/ajo-decisioning-target-tpid.png)
+
+In your mobile application, integrate the Identity for Edge Network extension to add the Target Third Party ID in the Identity Map in the personalization query request to the Edge network when calling the `updatePropositions` API. For more details see, [Identity for Edge Network - updateIdentities API](https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network/api-reference#updateidentities).
+ 
+{% tabs %}
+{% tab title="iOS - Swift" %}
+#### Swift
+
+```swift
+let identityMap = IdentityMap()
+identityMap.add(item: IdentityItem(id: "1111", authenticatedState: AuthenticatedState.authenticated, primary: true),
+                withNamespace: "userCRMID") // userCRMID being used as Third Party ID
+Identity.updateIdentities(with: identityMap)
+```
+
+#### Objective-C
+
+```objc
+AEPIdentityItem *item = [[AEPIdentityItem alloc] initWithId:@"1111" authenticatedState:AEPAuthenticatedStateAuthenticated primary:true];
+
+AEPIdentityMap *identityMap = [[AEPIdentityMap alloc] init];
+[identityMap addItem:item withNamespace:@"userCRMID"]; // userCRMID being used as Third Party ID
+
+[AEPMobileEdgeIdentity updateIdentities:identityMap];
+```
+{% endtab %}
+{% endtabs %}
+
+### Analytics for Target (A4T)
+
+Set up the Analytics for Target (A4T) cross-solution integration, by enabling the A4T campaigns, to use Analytics as the reporting source for an activity. Subsequently, all reporting and segmentation for that activity is based on Analytics data collection. For more information, see [Adobe Analytics for Adobe Target (A4T)](https://docs.adobe.com/content/help/en/target/using/integrate/a4t/a4t.html).
```

Comment:
We don't need the comma there. I'd just go with "Set up the ... integration by enabling the A4T campaigns to use Analytics..."